test cases/cython/4 limited api/test.json

```diff
@@ -0,0 +1,10 @@
+{
+  "matrix": {
+    "options": {
+      "cython_language": [
+         { "val": "c" },
+         { "val": "cpp" }
```

Comment:
Aside: is there anything interesting about handling C vs C++ here? I notice you are testing both (so, two tests not one).

Reply:
Ah good catch: this is a vestige from a previous attempt at making a valid test. I'll remove this, I don't think there is a difference between C and C++ here.